plugins/messagelayer/mana_plugin.go

```diff
@@ -404,15 +409,20 @@ func GetOnlineNodes(manaType mana.Type) (onlineNodesMana []mana.Node, t time.Tim
 	knownPeers = append(knownPeers, deps.Local.Peer)
 	onlineNodesMana = make([]mana.Node, 0)
 	for _, peer := range knownPeers {
-		if baseManaVectors[manaType].Has(peer.ID()) {
-			var peerMana float64
-			peerMana, t, err = baseManaVectors[manaType].GetMana(peer.ID())
-			if err != nil {
-				return nil, t, err
-			}
-			if peerMana > 0 {
-				onlineNodesMana = append(onlineNodesMana, mana.Node{ID: peer.ID(), Mana: peerMana})
-			}
+		if !baseManaVectors[manaType].Has(peer.ID()) {
+			continue
+		}
+		fmt.Println("Has mana", manaType.String(), peer.ID().String())
+		var peerMana float64
+		peerMana, t, err = baseManaVectors[manaType].GetMana(peer.ID())
+		if manaType == mana.AccessMana && peerMana < tangle.MinMana && err == nil {
```

Comment:
Did I understand this correctly that this is needed to make our hacky test-state work in which below-min-nodes are still allowed to issue transactions and that we cannot achieve this behavior by just changing `scheduler.go`?
In this case we must at least clearly mark this fact, e.g. add a TODO here, change the function description and/or create an issue pointing to this line.
Otherwise we will forget this and debug forever when the proper behavior is activated.